src/directives/if.js

```diff
@@ -1,4 +1,4 @@
-import { transitionIn, transitionOut } from '../utils'
+import { transitionIn, transitionOut } from '../transitions'
```

Comment:
do we need to apply the same fix we did in show? So transition won't overlap?

Reply:
I haven't went deeper than `x-show`. But i did make sure refactoring did not effect existing directives. Anyways since the remaining transitions are carried with `el`, we can run `el.transition_remaining()` anytime, anywhere, anyhow.